addons/rating/models/rating_mixin.py

```diff
@@ -78,6 +78,25 @@ def _compute_rating_stats(self):
             record.rating_count = mapping.get(record.id, {}).get('rating_count', 0)
             record.rating_avg = mapping.get(record.id, {}).get('rating_avg', 0)
 
+    def _search_rating_avg(self, operator, value):
+        if operator not in ['=', '!=', '<=', '<', '>', '>=']:
+            raise NotImplementedError('This operator %s is not supported in this search method.' % operator)
+        if not isinstance(value, (int, float)):
+            raise NotImplementedError('The value must be an integer or a float value.')
+        query = f"""
+            SELECT doc.id
+              FROM {self._table} as doc
+        INNER JOIN rating_rating R ON R.res_id = doc.id AND res_model = %s
+             WHERE R.consumed = 't' AND R.rating >= %s
+          GROUP BY doc.id
+            HAVING avg(R.rating) {operator} %s
+        """
+        return [(
+            'id',
+            'inselect' if operator != '!=' else 'not inselect',
+            (query, (self._name, RATING_LIMIT_MIN, value,))
+        )]
```

Comment:
I use the f-string for `self._table` and `operator` because with %s, the quotes are automatically added before and after a string. So since the operator hasn't to be interpreted as a string but as an SQL operator, I don't find another solution than directly inject the operator. 🙂 
For the `self._table`, I will recheck when I passed it with %s but I think I had an error. 😨